docs/react-testing-library.md

```diff
@@ -0,0 +1,72 @@
+# `react-testing-library` config
+
+Config for projects using **React Testing Library** for testing.
+
+## 🏗️ Setup
+
+1. If you haven't already, make sure to [install `@code-pushup/eslint-config` and its required peer dependencies](../README.md#🏗️-setup).
+2. Since this plugin requires additional peer dependencies, you have to install them as well:
+
+   ```sh
+   npm install -D eslint-plugin-testing-library
+   ```
+
+3. Add to your `eslint.config.js` file:
+
+   ```js
+   import react-testing-library from '@code-pushup/eslint-config/react-testing-library.js';
+   import tseslint from 'typescript-eslint';
+   
+   export default tseslint.config(
+     ...react-testing-library,
+     {
+       // customize rules if needed:
+       rules: {
+         // e.g. to enforce using `queryBy*` or `getBy*` for matchers:
+         'testing-library/prefer-query-matchers': [
+           'warn',
+           { validEntries: [{ matcher: 'toBeVisible', query: 'get' }] },
+         ]
+       }
+     }
```

Comment:
I don't think this example is very realistic. I try to keep the setup docs as simple as possible, and only include extra steps which are required or very common. We don't need to explain that rules can be customized in general.

Reply:
Okay. The options I selected don't appear in the documentation though, could be a bug?